js/noisemaker.js

```diff
@@ -0,0 +1,54 @@
+const notes = ['c4','d4','e4','f4','g4','a4','b4','c5'];
+const sounds = [];
+
+const soundContext = new AudioContext();
+
+function loadSound(filename, buffer, index) {
+  const request = new XMLHttpRequest();
+  request.open('GET', filename, true);
+  request.responseType = 'arraybuffer';
+
+  request.onload = () => {
+    soundContext.decodeAudioData(request.response, (newBuffer) => {
+      buffer[index] = newBuffer;
+      console.log(`${filename} loaded!`, newBuffer)
+    });
+  };
+
+  request.send();
+}
+
+function playSound(idx, options) {
```

Comment:
Your code is very clean, I like it. But I didn't find where would you use these options, and you aren't actually passing any also.

Reply:
You are right! 🤦‍♂️ To be honest, I took some code I used for another project I did ([Batteship board game](https://github.com/oscarnava/Battleship)) and after removing the unneeded code that parameter was left behind. I'll remove it. Thanks for pointing it out! 🙏